class Failure {
  final String message;
  const Failure([this.message = 'unexpected_error']);
}
